python/graphstorm/eval/evaluator.py

```diff
@@ -1034,64 +1141,21 @@ def _get_major_score(self, score):
             major_score = score[self.major_etype]
         return major_score
 
-    def evaluate(self, val_rankings, test_rankings, total_iters):
-        """ `GSgnnLinkPredictionTrainer` and `GSgnnLinkPredictionInferrer` will call this function
-        to compute validation and test mrr scores.
+    def get_val_score_rank(self, val_score):
+        """ Get the rank of the given ``major_etype`` validation score by comparing its value
```

Comment:
Who defines the ``major_etype``? How to get the ``major_etype``?

Reply:
This argument comes from the initialization. Users who construct this `GSgnnPerEtypeMrrLPEvaluator` object could give this argument.

This `get_val_score_rank` will call the `_get_major_score()` method to get this `major_etype`.

Reply:
Update to `Get the rank of the validation score of the given ``major_etype`` in class  initialization by ......` to make the description more clear.